src/dma/cli/main.py

```diff
@@ -166,36 +167,25 @@ def collect_data(
 
 async def _collect_data(
     console: Console,
-    db_type: Literal["POSTGRES", "MYSQL", "ORACLE", "MSSQL"],
-    username: str,
-    password: str,
-    hostname: str,
-    port: int,
+    src_info: SourceInfo,
     database: str,
     collection_identifier: str | None,
     working_path: Path | None = None,
 ) -> None:
-    async_engine = get_engine(db_type, username, password, hostname, port, database)
     working_path = working_path or Path("tmp/")
-    execution_id = f"{db_type}_{current_version!s}_{datetime.now(tz=timezone.utc).strftime('%y%m%d%H%M%S')}"
+    f"{src_info.db_type}_{current_version!s}_{datetime.now(tz=timezone.utc).strftime('%y%m%d%H%M%S')}"
```

Comment:
```suggestion
    _execution_id = f"{src_info.db_type}_{current_version!s}_{datetime.now(tz=timezone.utc).strftime('%y%m%d%H%M%S')}"
```

I'll need to review where this goes after we merge.  The linting will automatically remove variables that are unused and are not prefixed with `_`.  This just prevents it from being stripped from the file.